nntrainer/tensor/float_tensor.cpp

```diff
@@ -325,6 +325,56 @@ void FloatTensor::copyData(const TensorV2 &from) {
   }
 }
 
+int FloatTensor::multiply_i(float const &value) {
+  float *data = (float *)getData();
+  unsigned int len = size();
+
+  sscal(len, value, data, 1);
+
+  return ML_ERROR_NONE;
+}
+
+TensorV2 &FloatTensor::multiply(float const &value, TensorV2 &out) const {
+  auto f = std::bind(std::multiplies<float>(), std::placeholders::_1, value);
+  apply(f, out);
+  return out;
+}
+
+TensorV2 &FloatTensor::multiply(TensorV2 const &m, TensorV2 &output,
+                                const float beta) const {
+  auto f = [&](const BroadcastInfoV2 &e, const float *buf, const float *m_buf,
+               float *out_buf) {
+    if (e.strides[3] == 1 && output.getStrides()[3] == 1 && strides[3] == 1 &&
+        beta == 0.0) {
+      std::transform(buf, buf + e.buffer_size, m_buf, out_buf,
```

Comment:
Even though we use this std::transform, we haven't tested it with other libs or implementations. Not in this PR, but we have to evaluate it.

Reply:
gotcha. I'll add some testing to verify.